third-party/realdds/include/realdds/dds-metadata-syncer.h

```diff
@@ -70,9 +70,9 @@ class dds_metadata_syncer
     std::deque< key_metadata > _metadata_queue;
     std::mutex _queues_lock;
 
-    on_frame_release_callback _on_frame_release = nullptr;
-    on_frame_ready_callback _on_frame_ready = nullptr;
-    on_metadata_dropped_callback _on_metadata_dropped = nullptr;
+    on_frame_release_callback _on_frame_release;
```

Comment:
Why did you remove the initialization?
Even if it is the same, explicit is better then implicit :-)

Reply:
I agree unless it's obvious. Anyway if you decide to do something different in the ctor then this will be in conflict. I'd rather people went to the ctor to check (unless there none).